lib/library.rb

```diff
@@ -0,0 +1,56 @@
+require 'date'
+require 'pry'
+
+class Library
+  attr_accessor :books #TODO duedate person status
+
+  def initialize(options={})
+      @books = options[:books] || []
+  end
+
+  def check_in(book)
+      add = {title: book[:title],
+           author: book[:author],
+           status: :okay,
+           person: :non,
+           duedate: " "}
+     @books.push add
+  end
+
+  def lend(user, args={})
+      book = @books.detect { |obj| obj[args.keys.first] == args[args.keys.first] }
```

Comment:
Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)